docs/mobile-apps/live-testing/live-mobile-app-testing.md

```diff
@@ -216,12 +216,15 @@ This feature has a constraint on the maximum allowable length of the test name,
 | Icon                                                                                                 | Name         | Description                                                                                                                                                                                             |
 | :--------------------------------------------------------------------------------------------------- | :----------- | :------------------------------------------------------------------------------------------------------------------------------------------------------------------------------------------------------ |
 | <img src={useBaseUrl('img/live-testing/log-level.png')} alt="Log Level" width="85"/>                 | Log Level    | <ul><li>VERBOSE </li><li>DEBUG </li><li>INFO </li><li>WARN </li><li>ERROR </li><li>ASSERT </li></ul>                                                                                                    |
+| <img src="https://user-images.githubusercontent.com/68342451/236323299-3e75d296-a6ec-4130-b131-e9c901c43429.png">                 | [App Log](/mobile-apps/features/mobile-app-diagnostics/app-logs/)    | <ul><li>VERBOSE </li><li>DEBUG </li><li>INFO </li><li>WARN </li><li>ERROR </li><li>ASSERT </li></ul>                                                                                                    |
```

Comment:
The image should be added to the `static/img` folder and the img src (`<img src="https://user-images.githubusercontent.com/68342451/236323299-3e75d296-a6ec-4130-b131-e9c901c43429.png">`) should be updated.